ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -2247,6 +2247,89 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-agent-compact-ipv4-none-platform,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build05
+    context: ci/prow/e2e-agent-compact-ipv4-vmware
+    decorate: true
+    decoration_config:
+      timeout: 6h0m0s
+    labels:
+      ci-operator.openshift.io/cloud: equinix-ocp-metal
+      ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
+      ci-operator.openshift.io/cluster: build05
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-master-e2e-agent-compact-ipv4-vmware
+    optional: true
+    rerun_command: /test e2e-agent-compact-ipv4-vmware
+    run_if_changed: /agent/
```

Comment:
I think we decided to take this out, but once it is in the job is gets preserved even if you take it out of the config. (In the old days, you couldn't even put this stuff in the config, and many jobs still haven't so we have to continue preserving the data in the job because it's the only place it is recorded in many cases.) It pays to delete the job from this file and regenerate it to make sure it matches the config.